### ISSUE TITLE
how to train a machine learning model for text classification using Python 

### ISSUE THREAD

**Issue by Samuelliputra** [Samuelliputra/MLusing-Python#1]
# Import the necessary libraries
from sklearn.feature_extraction.text import CountVectorizer
from sklearn.naive_bayes import MultinomialNB
from sklearn.pipeline import Pipeline

# Define the text classifier
text_classifier = Pipeline([
    ('vectorizer', CountVectorizer()),
    ('classifier', MultinomialNB())
])

# Train the classifier on the training data
text_classifier.fit(X_train, y_train)

# Predict labels for the test data
predicted = text_classifier.predict(X_test)

# Calculate the accuracy of the predictions
accuracy = (predicted == y_test).mean()
print(f'Test accuracy: {accuracy:.2f}')

#Now How to make the ML Learn all system and automatically grow up The AI it by the self
